packages/vue/src/components/result/ResultList.jsx

```diff
@@ -0,0 +1,113 @@
+import { RLConnected } from './ReactiveList.jsx';
+import Title from '../../styles/Title';
+import VueTypes from 'vue-types';
+import ListItem, { container, Image } from '../../styles/ListItem';
+import types from '../../utils/vueTypes.js';
+import { helper } from '@appbaseio/reactivecore';
+
+const { getClassName } = helper;
+
+const ResultList = {
+	name: 'ResultList',
+	props: {
+		currentPage: VueTypes.number.def(0),
+		includeFields: types.includeFields.def(['*']),
+		// component props
+		className: types.string,
+		componentId: types.stringRequired,
+		dataField: types.stringRequired,
+		defaultQuery: types.func,
+		excludeFields: types.excludeFields.def([]),
+		innerClass: types.style,
+		listClass: VueTypes.string.def(''),
+		loader: types.title,
+		onAllData: types.func,
+		onData: types.func,
+		onResultStats: types.func,
```

Comment:
We also support scopedSlot for it ( `onResultStats` ) in ReactiveList, so remember to pass it down to ReactiveList.